middleware-catalog-module/.idea/compiler.xml

```diff
@@ -8,25 +8,34 @@
         <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
```

Comment:
This files inside .idea directory should not be here. They are on .ignore. You need to delete them in your machine then make a git add, git commit, git push.